samples/MvcSample.Web/Monitoring/MonitoringMiddlware.cs

```diff
@@ -73,7 +73,7 @@ private string GetTypeName(Type type)
             var name = type.Name;
             var isArray = false;
 
-            if (typeof(Array).IsAssignableFrom(type))
+            if (typeof(Array).GetTypeInfo().IsAssignableFrom(type.GetTypeInfo()))
```

Comment:
minor but doesn't the original line compile fine once you've added the `using System.Reflection`?